chart/templates/workers/worker-deployment.yaml

```diff
@@ -111,7 +110,7 @@ spec:
           command:
             - chown
             - -R
-            - "{{ .Values.uid }}:{{ .Values.gid }}"
+            - "{{ include "globalSecurityContextIds" (list . .Values.workers) }}"
```

Comment:
I'd wondering if we'd be better off doing something like this instead (untested, but you get the idea):

`bash -c 'exec chown -R $(id -u):$(id -g) {{ template "airflow_logs" . }}`

Then `chown` can just use the uid/gid it's running as instead of being configured explicitly.